### PR TITLE
refactor(core): move concrete-commons content into core

### PR DESCRIPTION
**Pull request by sarah-ek**
### Description

merges concrete-commons into concrete-core

### Checklist 

(Use '[x]' to check the checkboxes, or submit the PR and then click the checkboxes)

* ~[ ] Tests for the changes have been added (for bug fixes / features)~
* ~[ ] Docs have been added / updated (for bug fixes / features)~
* [x] The PR description links to the related issue (to link an issue, use '#XXX'.)
* [x] The tests on AWS have been launched and are successful (comment with @slab-ci cpu_test and/or @slab-ci gpu_test to trigger the tests)
* [x] The draft release description has been updated
* [x] Check for breaking changes (including serialization changes) and add them to commit message following the conventional commit [specification][conventional-breaking]

### CODE REVIEW

**Comment by agnesLeroy** [comment]
@sarah-ek it seems there are still some mentions to concrete-commons in concrete-tasks :slightly_smiling_face:

**Comment by agnesLeroy** [comment]
May I ask you to add the mention BREAKING CHANGE in your commit description as well please? We're trying to have this practice in place to better handle version updates (even though I'm not sure this is super useful at the moment).

**Comment by sarah-ek** [comment]
i edited the commons references in concrete-tasks. and changed the commit description

**Comment by IceTDrinker** [comment]
@slab-ci cpu_test

**Comment by sarah-ek** [comment]
@slab-ci cpu_test

File: concrete-core-fixture/src/fixture/cleartext_creation.rs [hunk 1]
```diff
@@ -4,7 +4,7 @@ use crate::generation::synthesizing::SynthesizesCleartext;
 use crate::generation::{IntegerPrecision, Maker};
 use crate::raw::generation::RawUnsignedIntegers;
 use crate::raw::statistical_test::assert_noise_distribution;
-use concrete_commons::dispersion::Variance;
+use concrete_core::concrete_commons::dispersion::Variance;
```
Comment (agnesLeroy) [inline concrete-core-fixture/src/fixture/cleartext_creation.rs:7]: For all concrete-core imports we use the prelude usually, that reexposes all available public structures/traits/functions.

**Review by agnesLeroy — COMMENTED** [review]
Hello @sarah-ek! Thanks a lot for this PR: looking at it I realize concrete-core sources would probably look more organized if:
- dispersion
- key_kinds
- parameters
were placed in the `specification` module, and
`numeric` could be moved to the private `commons` module.
How does that sound to you?

I wrote it in a comment after, but we re-expose all concrete-core entry points in the prelude so users don't need to know anything about the internal structure of concrete-core to import something: may I ask you to do the same for dispersion/key_kinds/parameters please?

**Comment by sarah-ek** [comment]
@slab-ci cpu_test

**Comment by agnesLeroy** [comment]
@slab-ci gpu_test

**Review by agnesLeroy — APPROVED** [review]

**Review by IceTDrinker — APPROVED** [review]
Not much to say here, tests are passing, files look to be in the right places 👍 thanks a lot for the effort

**Pull request merged**